contracts/exchange/test/transactions.ts

```diff
@@ -157,6 +159,44 @@ describe('Exchange transactions', () => {
         taker2TransactionFactory = new TransactionFactory(taker2PrivateKey, exchangeInstance.address, chainId);
     });
     describe('executeTransaction', () => {
+        describe('general functionality', () => {
+            it('should log the correct transactionHash if successfully executed', async () => {
+                const order = await orderFactory.newSignedOrderAsync();
+                const orders = [order];
+                const data = exchangeDataEncoder.encodeOrdersToExchangeData(ExchangeFunctionName.FillOrder, orders);
+                const transaction = await takerTransactionFactory.newSignedTransactionAsync({ data });
+                const transactionReceipt = await exchangeWrapper.executeTransactionAsync(transaction, takerAddress);
+                const transactionExecutionLogs = transactionReceipt.logs.filter(
+                    log =>
+                        (log as LogWithDecodedArgs<ExchangeTransactionExecutionEventArgs>).event ===
+                        'TransactionExecution',
+                );
+                expect(transactionExecutionLogs.length).to.eq(1);
+                const executionLogArgs = (transactionExecutionLogs[0] as LogWithDecodedArgs<
+                    ExchangeTransactionExecutionEventArgs
+                >).args;
+                expect(transactionHashUtils.getTransactionHashHex(transaction)).to.equal(
```

Comment:
Nit: Assertion order is backwards: should be `actual.equal(expected)`. Also appears in other places in this file.